src/Component.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+class Component extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+            count: 0
+        }
+
+        this.handleClick = this.handleClick.bind(this);
+    }
+
+    handleClick() {
+        this.setState({
+            count: this.state.count + 1
+        });
+    }
+
+    render() {
+        return (
+            <div style={{margin: '5px', padding: '5px'}}>
```

Comment:
You can write just numbers if you need pixels in JSX styles:
```
            <div style={{margin: 5, padding: 5 }}>
```

Reply:
Thank you. That's super helpful to know!